libs/remix-ui/solidity-compile-details/src/lib/solidity-compile-details.tsx

```diff
@@ -13,6 +14,10 @@ export interface RemixUiCompileDetailsProps {
   saveAs: any
 }
 
+export type ContractPropertyName = 'compilerInput' | 'name' | 'metadata' | 'bytecode' | 'abi' | 'storageLayout'
```

Comment:
this type is already define below, we should have one declaration. I know this add a dependency, but that's probably  better than duplicating the types.